lib/gr-client.js

```diff
@@ -0,0 +1,53 @@
+const Grenache = require('grenache-nodejs-http')
+const Link = require('grenache-nodejs-link')
+const conf = require('../config/moonbeam.conf')
+
+const Peer = Grenache.PeerRPCClient
+
+const link = new Link(conf.link)
+link.start()
+
+const peer = new Peer(link, {})
+peer.init()
+
+exports.notifySignup = ({ user }, cb) => {
```

Comment:
you can use https://github.com/bitfinexcom/bfx-lib-server-js

Reply:
will do, thanks